selectors/emojis.ts

```diff
@@ -56,6 +58,14 @@ export const getOneClickReactionEmojis = createSelector(
             return [];
         }
 
-        return recentEmojis.map((recentEmoji) => emojiMap.get(recentEmoji)).filter(Boolean).slice(-3).reverse();
+        const sortedRecentEmojis = recentEmojis.sort(
```

Comment:
Do you think it would make any difference performance-wise if we stored a sorted array of emoji to begin with instead of sorting it here? I could go either way on that since I don't imagine it'll make a difference in the long run.

That said, there's a bug here that might show up, particularly if `recentEmojis` is memoized like we should be doing. `recentEmojis.sort(...)` actually mutates `recentEmojis`, so we should copy `recentEmojis` before sorting it